akka-stream/src/main/scala/akka/stream/javadsl/SubFlow.scala

```diff
@@ -607,6 +607,25 @@ class SubFlow[-In, +Out, +Mat](delegate: scaladsl.SubFlow[Out, Mat, scaladsl.Flo
     new SubFlow(delegate.recover(pf))
 
   /**
+    * RecoverWith allows to switch to alternative Source on flow failure. It can switch
+    * several times by materializing each time Source that `pf` returns.
```

Comment:
Funky english here, what about: "It will stay in effect after a failure has been recovered so that each time there is a failure it is fed into the `pf` and a new Source may be materialized."?